docs/sources/flow/reference/components/otelcol.exporter.prometheus.md

```diff
@@ -48,9 +49,8 @@ OpenTelemetry instrumentation scopes are converted into `otel_scope_info`
 metrics. Set the `include_scope_info` and `include_target_info` arguments to
 `false`, respectively, to disable the custom metrics.
 
-When `include_scope_info` is `true`, the instrumentation scope name and version
-are added as `otel_scope_name` and `otel_scope_version` labels to every
-converted metric sample.
+`include_scope_labels` is used to enable including the `otel_scope_name` and
+`otel_scope_version` labels to every converted metric sample.
```

Comment:
```suggestion
When `include_scope_labels` is `true`  the `otel_scope_name` and
`otel_scope_version` labels are added to every converted metric sample.
```

I am guessing that you need to set this new arg to true?

Reply:
ya that sounds better